site/source/tutorials/002_creating_an_application/index.md

```diff
@@ -29,7 +29,11 @@ HTML pages are the foundation for every web application and Dojo 2 applications
 
 {% include_codefile 'demo/initial/biz-e-corp/src/main.ts' line:4 %}
 
-Notice that we are searching for the `my-app` element and assigning it to the constant `root`. The application is using this node to determine where to place the Dojo 2 application on the page. Everything that the application does should be contained within this single element. There are several benefits to this approach. First, a Dojo 2 application can easily coexist on a page with other content. That content can consist of static assets, a legacy application or even another Dojo 2 application. The next advantage is that Dojo 2 can leverage third-party libraries with ease. For example, if you would like to use the [moment.js](https://momentjs.com/) library to simplify working with time in your application, it can be loaded in the main HTML document, and the Dojo 2 application can take advantage of it.
+Notice that we are searching for the `my-app` element and assigning it to the constant `root`. The application is using this node to determine where to place the Dojo 2 application on the page. Everything that the application does should be contained within this single element. There are several benefits to this approach:
+
+* A Dojo 2 application can easily coexist on a page with other content.
+* That content can consist of static assets, a legacy application or even another Dojo 2 application.
+* Dojo 2 can leverage third-party libraries with ease. For example, if you would like to use the [moment.js](https://momentjs.com/) library to simplify working with time in your application, it can be loaded in the main HTML document, and the Dojo 2 application can take advantage of it.
```

Comment:
I know this already existed, but do you think we should just remove it (or re-word it so it's accurate)? Because a consumer shouldn't need to include these libraries in the HTML